open_instruct/utils.py

```diff
@@ -639,7 +653,7 @@ def parse_yaml_and_args(self, yaml_arg: str, other_args: Optional[List[str]] = N
 
         return outputs
 
-    def parse(self) -> DataClassType | Tuple[DataClassType]:
+    def parse(self) -> Union[DataClassType, Tuple[DataClassType]]:
```

Comment:
Why this needed?

Reply:
Friendlier for python 3.9. This syntax is only supported at python 3.10+